### ISSUE TITLE
Have trouble using Skeletal animation

### ISSUE THREAD

**Issue by zhangweizhe** [Samsung/GearVRf#1375]
In the [gvr-meshanimation](https://github.com/gearvrf/GearVRf-Demos/tree/master/gvr-meshanimation) , the skeletal mesh,  animation sequence and the information of the texture t_rex_texture_diffuse.png  are all stored in  TRex_NoGround.fbx. 
Now I have a fbx file [plane.fbx](https://github.com/zhangweizhe/hello-world/blob/master/plane.fbx) including skeletal mesh and animation sequence, but without any information about the texture.
I'd like to create a GVRSceneObject, then apply the skeletal mesh and animation sequence  loaded from the fbx file to the sceneObject, and then load image as texture for the sceneObject.
Here are the codes I write

public class AnimationMain extends GVRMain{

    private GVRScene mScene;
    @Override
    public void onInit(GVRContext gvrContext) throws Throwable {
        super.onInit(gvrContext);
        mScene = gvrContext.getMainScene();
        mScene.setBackgroundColor(0.8f, 0.8f, 0.8f, 1);
        
        GVRMesh mesh = gvrContext.getAssetLoader().loadMesh(new GVRAndroidResource(gvrContext, "plane.fbx"));
        GVRTexture texture = gvrContext.getAssetLoader().loadTexture(new GVRAndroidResource(gvrContext, R.drawable.gearvr_logo));
        GVRSceneObject animaObj = new GVRSceneObject(gvrContext, mesh, texture);
        animaObj.getTransform().setPosition(0,0,-3);

        mScene.addSceneObject(animaObj);
    }
}

It doesn't work as gvr-meshanimation, the sceneobject is immobile. 
In other words,  I want to load the skeletal animation from fbx file and the texture dividually from drawable, and attach them to a GVRSceneObject. How can I do that?

**Comment by apjagdale** [comment]
Hi,
You can do it like this:

`GVRSceneObject animaObj = gvrContext.getAssetLoader().loadModel(mModelPath, mainScene);
            GVRTexture texture = gvrContext.getAssetLoader().loadTexture(new GVRAndroidResource(gvrContext, R.drawable.gearvr_logo));`
`ArrayList<GVRRenderData> renderDatas = animaObj.getAllComponents(GVRRenderData.getComponentType());
            renderDatas.get(0).getMaterial().setTexture("diffuseTexture", texture);`

**Comment by zhangweizhe** [comment]
Hello @apjagdale , thank you for your reply.
I'm sorry that it does't work as I except, I can't see anything in the scene
Here are the codes
`
public class AnimationMain extends GVRMain {
    private GVRScene mainScene;
    @Override
    public void onInit(GVRContext gvrContext) throws Throwable {
        super.onInit(gvrContext);
        mainScene = gvrContext.getMainScene();
        mainScene.setBackgroundColor(0.8f, 0.8f, 0.8f,1);

        GVRSceneObject aniObj = gvrContext.getAssetLoader().loadModel("plane.fbx", mainScene);
        GVRTexture texture = gvrContext.getAssetLoader().loadTexture(new GVRAndroidResource(gvrContext, R.drawable.gearvr_logo));
        ArrayList<GVRRenderData> renderDatas = aniObj.getAllComponents(GVRRenderData.getComponentType());
        renderDatas.get(0).getMaterial().setTexture("diffuseTexture", texture);
        aniObj.getTransform().setPosition(0,0,-3);

        mainScene.addSceneObject(aniObj);
    }
}
`

**Comment by apjagdale** [comment]
Hi @zhangweizhe 
Your plane is too huge. I placed it at a position of (0,0, -100). I could also see the animation similar to heart beat.
SetMainTexture sets u_texture whereas setTexture is used to set a specific texture.

**Comment by zhangweizhe** [comment]
Hello @apjagdale 
Yes,  I can see the animated logo!!
Thank you for your reply, thank you so much!

**Comment by liaxim** [comment]
@zhangweizhe Can we consider the issue closed then? Thanks.
